doc/user_guide/transform/aggregate.rst

```diff
@@ -70,3 +159,39 @@ class, which has the following options:
 The :class:`~AggregatedFieldDef` objects have the following options:
 
 .. altair-object-table:: altair.AggregatedFieldDef
+
+.. _agg-func-table:
+
+List of Aggregation Functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to ``count`` and ``average``, there are a large number of available
+aggregation functions built into Altair; they are listed in the following table:
+
+=========  ===========================================================================  =====================================
+Aggregate  Description                                                                  Example
+=========  ===========================================================================  =====================================
```

Comment:
The vega-lite [docs](https://vega.github.io/vega-lite/docs/aggregate.html#ops) appear to list these in a more logical (if implicit) order,  starting with count-related functions (including `count`, `valid`, `values`, `missing`, and `distinct`), moving to basic mathematical operations (`sum`, `product`), then to central tendency measures (`mean`/`average`, `variance`/`variancep`, `stdev`/`stdevp`, `stderr`, `median`), followed by distribution statistics (`q1`, `q3`, `ci0`, `ci1`), and finally ending with range functions (`min`/`argmin`, `max`/`argmax`). The ordering here appears to be in alphabetial order, though it's not strictly so (e.g. `ci01`). I would have a slight preference for the vega-lite-style functional organization scheme (and with explicit headings for the categories).

Reply:
I agree on changing the order.

I'd probably need to see the end result of adding categories though.
The naive approach of just adding a category field would add a lot of repetition